flexget/components/emby/api_emby.py

```diff
@@ -373,12 +378,12 @@ def check_token_data(self, token_data, login_type):
 
         self._userid = token_data.get('userid')
         self._token = token_data.get('token')
-        self._login_type = token_data.get('type')
+        self._login_type = token_data.get('login_type')
         self._connect_username = token_data.get('connect_username', '')
         self.host = token_data.get('host', '')
         self._logged = True
         endpoint = EMBY_ENDPOINT_USERINFO.format(userid=token_data['userid'])
-        response = EmbyApi.resquest_emby(endpoint, self, 'GET')
+        response = EmbyApi.resquest_emby(endpoint, self, 'GET', plugin_error=False)
```

Comment:
Would it be nicer to just catch `PluginError` here, rather than adding another keyword argument?

Reply:
I need the login requested to return false, so that I'm able to try to make a login again! For example if the ip on emby connect changed! Probably you are asking why not use a try catch on this... If that's the case, you probably are right! That would be a better approach! I'll change it.